corgi/collectors/brew.py

```diff
@@ -588,8 +590,8 @@ def get_container_build_data(self, build_id: int, build_info: dict[str, Any]) ->
 
         # During collection we are only able to inspect docker config labels on
         # attached arch specific archives. We do this loop here to save the description, license,
-        # name label, and repository url also on the index container object at the root of the tree.
-        for attr in ("description", "license", "name_from_label", "repository_url"):
+        # also on the index container object at the root of the tree.
+        for attr in ("name", "description", "license", "name_from_label", "repository_url"):
```

Comment:
Looks like a typo here, where you accidentally removed part of the comment? Or maybe that was on purpose, but you forgot to change the code to match?
```suggestion
        # name, label, and repository_url also on the index container object at the root of the tree.
        for attr in ("name", "description", "license", "name_from_label", "repository_url"):
```
If you make my suggested change above, I guess `name` should become `name_and_label`.